functions-and-operators/json-functions.md

```diff
@@ -103,6 +98,8 @@ The following JSON functions are unsupported in TiDB. You can track the progress
 
```

Comment:
I don't think this renders correctly, please format this whole paragraph as the two first bullet points.

Reply:
I think the context of your comment got lost somehow. It also looks like it is already fixed. Could you check if this is really the case and resolve this discussion?